pennylane_qiskit/qiskit_device2.py

```diff
@@ -622,8 +610,9 @@ def _process_estimator_job(measurements, job_result):
         along with some metadata. Extract the relevant number for each measurement process and
         return the requested results from the Estimator executions."""
 
-        expvals = job_result.values
-        variances = [res["variance"] for res in job_result.metadata]
+        expvals = [res.data.evs.item() for res in job_result]
+        variances = [res.data.stds.item()**2*4096 for res in job_result] # this 4096 is the # of shots
+        # ToDo: Track the # of shots and use that to calculate the variance
```

Comment:
Qiskit removed the information for variances, opting to give `stds` instead. This means we have to reverse engineer `var` from `stds`, which requires the # of shots ran. For now I'm using their default # of shots because the options (and by extension shots) processing will be done in a separate PR.